app/javascript/packages/document-capture/components/_file-input.scss

```diff
@@ -17,7 +17,7 @@
   border-width: 3px;
 }
 
-usa-file-input:not(
+.usa-file-input:not(
```

Comment:
What exactly are these styles meant to do? I'm having trouble figuring out what combination of states this is meant to apply to.

Reply:
It is supposed to apply to the acuant input fields that don't have values pending and are selfie inputs (not front/back ID inputs)

Reply:
During initial capture and all subsequent selfie input fields during recapture if failed attempts.

Reply:
The image ratio changes (21rem height) should only apply to the selfie capture. Below are some images of the ratio in effect. The ID fields should remain unchanged.
<img width="816" alt="Screenshot 2024-10-04 at 12 11 54 PM" src="https://github.com/user-attachments/assets/aef9f5b0-e4fd-4c56-8754-b084ab3eeba1">
<img width="831" alt="Screenshot 2024-10-04 at 12 12 04 PM" src="https://github.com/user-attachments/assets/e24d4290-061a-4088-a099-6e52efa2678b">

<img width="401" alt="Screenshot 2024-10-04 at 12 12 25 PM" src="https://github.com/user-attachments/assets/29260235-a981-4a5c-87a3-7450cc426f4d">

<img width="654" alt="Screenshot 2024-10-04 at 12 11 44 PM" src="https://github.com/user-attachments/assets/0abf5481-b5bd-4d53-aca8-4b747d7441e5">

Reply:
Pushed up some edits to remove unneeded classes

Reply:
I don't think these styles are being applied. The screenshot you showed is not 21rem height.

This is what it would look like if the styles were actually applied. I simulated this by hard-coding a style for 21rem height. It looks different from your screenshot, and also overlaps with the button.

![image](https://github.com/user-attachments/assets/03d0f926-a6fc-4fcd-9445-b5dbf6fce172)